viz_config-example.yml

```diff
@@ -105,82 +104,20 @@ cities:
     dist: 0.007 # relative (fraction of window width) distance from dot to label
   pop_min: 500000 # minimum population size of cities to be chosen by default; `custom` will make additions or modifications and `exclude` will make subtractions
   custom: # cities to add or modify relative to the maps::us.cities dataset. added cities can use lat/long from us.cities if they are present in us.cities, even if they don't meet the pop_min criterion
-    #-
-    #  name: "Richmond VA"
-    #  text_angle: 170
-    -
-      name: "Raleigh NC"
-      text_angle: 170
-    -
-      name: "Charlotte NC"
-      text_angle: 170
     -
-      name: "Columbia SC"
-      text_angle: 170
-    #-
-    #  name: "Savannah GA"
-    #  text_angle: 170
-    #-
-    #  name: "Jacksonville FL"
-    #-
-    #  name: "Virginia Beach VA"
-    #  dot_cex: 0.8
-    #  text_cex: 0.8
-    #-
-    #  name: "Nags Head NC"
-    #  lat: 35.9425
-    #  long: -75.6228
-    #  dot_cex: 0.8
-    #  text_cex: 0.8
+      name: "Houston TX"
     -
-      name: "New Bern NC"
-      news: "CBS"
-      dot_cex: 0.8
-      text_cex: 0.8
-      lat: 35.107686
-      long: -77.051366
-    -
```

Comment:
I don't remember how we're managing multiple hurricanes within this repo - different branches, maybe? - but It might be nice to keep some of these customization examples in here if this is the template for future hurricane vizzies.